src/js/components/Friends/FriendInvitationEmailLinkDisplayForList.jsx

```diff
@@ -146,16 +146,16 @@ class FriendInvitationEmailLinkDisplayForList extends Component {
   }
 }
 FriendInvitationEmailLinkDisplayForList.propTypes = {
-  invitation_status: PropTypes.string, // Comes friend data object from API server
-  linked_organization_we_vote_id: PropTypes.string,
-  mutual_friends: PropTypes.number,
-  positions_taken: PropTypes.number,
+  invitationStatus: PropTypes.string, // Comes friend data object from API server
```

Comment:
Please make sure to also update these incoming props to camelCase starting at line 45:
```
    const {
      invitation_status: invitationState,
      mutual_friends: mutualFriends,
      positions_taken: positionsTaken,
      voter_twitter_handle: voterTwitterHandle,
      voter_email_address: voterEmailAddress,
      voter_photo_url_large: voterPhotoUrlLarge,
    } = this.props;
```